base/float.jl

```diff
@@ -727,14 +727,14 @@ end
     typemax(::Type{Float64}) = $(Inf64)
     typemin(x::T) where {T<:Real} = typemin(T)
     typemax(x::T) where {T<:Real} = typemax(T)
-
-    floatmin(::Type{Float16}) = $(bitcast(Float16, 0x0400))
-    floatmin(::Type{Float32}) = $(bitcast(Float32, 0x00800000))
-    floatmin(::Type{Float64}) = $(bitcast(Float64, 0x0010000000000000))
+    
     floatmax(::Type{Float16}) = $(bitcast(Float16, 0x7bff))
     floatmax(::Type{Float32}) = $(bitcast(Float32, 0x7f7fffff))
     floatmax(::Type{Float64}) = $(bitcast(Float64, 0x7fefffffffffffff))
-
+    for T in (Float16, Float32, Float64)
+        @eval floatmin(T) = nextfloat(zero(T))/eps(T)
```

Comment:
This would need to be `::Type{$T}`, and `$T` on the other uses. But why not just keep the hard-coded values?

Reply:
I suppose I misunderstood [this comment](https://github.com/JuliaLang/julia/issues/30387#issuecomment-447958221). I made the change only in Bigfloat. Is this what is expected?

Sorry for the trouble.